packages/p2p/src/components/buy-sell/popup.jsx

```diff
@@ -195,70 +89,71 @@ class Popup extends Component {
-
+                                        <div className='buy-sell__popup-field_wrapper'>
+                                            <Field name='amount'>
+                                                {({ field }) => (
+                                                    <Input
+                                                        {...field}
+                                                        data-lpignore='true'
+                                                        type='number'
+                                                        error={errors.amount}
+                                                        label={localize('Amount')}
+                                                        hint={
+                                                            <Localize
+                                                                i18n_default_text='Limits: {{min}}–{{max}} {{currency}}'
+                                                                values={{
+                                                                    min     : ad.min_transaction,
+                                                                    max     : ad.max_transaction,
+                                                                    currency: ad.offer_currency,
+                                                                }}
+                                                            />
+                                                        }
+                                                        className='buy-sell__popup-field'
+                                                        trailing_icon={<span className='buy-sell__popup-field--trailing'>{ad.offer_currency}</span>}
+                                                        onChange={(e) => {
+                                                            const amount = isNaN(e.target.value) ? 0 : e.target.value;
+                                                            this.setState({
+                                                                total_amount: this.getTotalAmount(amount),
+                                                            });
+                                                            handleChange(e);
+                                                        }}
+                                                        required
+                                                    />
+                                                )}
+                                            </Field>
+                                            <div className='buy-sell__popup-field'>
+                                                <span className='buy-sell__popup-info--title'>{ad.type === 'buy' ? localize('You send') : localize('You receive')}</span>
+                                                <p className='buy-sell__popup-info--text buy-sell__popup-info--strong'>{this.state.total_amount}{' '}{ad.transaction_currency}</p>
```

Comment:
we can use <strong> instead of adding more classes

Reply:
I think strong doesn't work. I can try.